tox.ini

```diff
@@ -14,10 +14,21 @@ commands =
     {toxinidir}/scripts/test/detect_missing_migrations.sh
     {toxinidir}/scripts/test/no_auto_migrations.sh
 
-passenv = *
+passenv =
+    COVERAGE_DIR
+    DATABASE_URL
+    BROKER_URL
+    CELERY_RESULT_BACKEND
+    PORT
+    TRAVIS*
+    CI
 setenv =
+    ELASTICSEARCH_INDEX=testindex
```

Comment:
What's this for?

Reply:
It's from micromasters, a copy and paste error